repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceConfig.java

```diff
@@ -103,9 +108,34 @@ public int getConnectionPoolSize()
         return connectionPoolSize;
     }
 
+    public void setIssuerUrl(String issuerUrl)
+    {
+        this.issuerUrl = issuerUrl;
+    }

```

Comment:
This should be right below the `getIssuerUrl` method as they belong to the same theme.

Reply:
Done.